clients/src/test/java/org/apache/kafka/clients/ClientUtilsTest.java

```diff
@@ -61,7 +61,7 @@ public void testParseAndValidateAddressesWithReverseLookup() {
         assertTrue(validatedAddresses.size() >= 1, "Unexpected addresses " + validatedAddresses);
         List<String> validatedHostNames = validatedAddresses.stream().map(InetSocketAddress::getHostName)
                 .collect(Collectors.toList());
-        List<String> expectedHostNames = asList("93.184.216.34", "2606:2800:220:1:248:1893:25c8:1946");
+        List<String> expectedHostNames = asList("93.184.215.14", "2606:2800:220:1:248:1893:25c8:1946");
```

Comment:
That is weird. What I see on my local is `93.184.215.14, 2606:2800:21f:cb07:6820:80da:af6b:8b2c`.

Reply:
Yeah, it's changing back n forth.

Reply:
Sorry, I read too quickly just now. Indeed, the IPv6 address I saw in tests is also 2606:2800:21f:cb07:6820:80da:af6b:8b2c.